document/src/main/java/com/ritense/document/service/DocumentService.java

```diff
@@ -62,4 +65,8 @@ public interface DocumentService {
     boolean currentUserCanAccessDocument(Document.Id documentId);
 
     void assignUserToDocument(UUID documentId, String assigneeId);
+
+    Set<String> getDocumentRoles(Document.Id documentId);
+
+    List<ManageableUser> getCandidateUsers(Document.Id documentId);
```

Comment:
`ManageableUser` contains properties like `password` and `activated`. It would be nicer if we had an object with the bare minimum that's not build for user management,.

Reply:
Changed